backend/tests/clubs/test_serializers.py

```diff
@@ -0,0 +1,152 @@
+from datetime import timedelta
+from unittest import mock
+
+from django.conf import settings
+from django.contrib.auth.models import Permission, User
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework.exceptions import ValidationError
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+
+from clubs.models import Club, Membership
+from clubs.serializers import ClubSerializer
+
+
+class ClubSerializerTestCase(TestCase):
+    def setUp(self):
+        self.factory = APIRequestFactory()
+        self.user = User.objects.create_user(username="testuser", password="12345")
+        self.club = Club.objects.create(name="Test Club", code="test", approved=True)
+
+        # Set renewal period in settings to current time
+        now = timezone.now()
+        self.renewal_start = now - timedelta(days=30)
+        self.renewal_end = now + timedelta(days=30)
+        settings.RENEWAL_PERIOD = (self.renewal_start, self.renewal_end)
+
+        self.serializer = ClubSerializer()
+
+    def test_privileged_user_sees_current_version(self):
+        self.user.user_permissions.add(
+            Permission.objects.get(codename="see_pending_clubs")
+        )
+        request = self.factory.get(reverse("clubs-detail", args=[self.club.code]))
```

Comment:
We don't want to actually send the request to the API -- we just want to create the request object, and then pass it to the serializer. This lets us test the serializer itself instead of the entire view. 

The `user_permissions.add` pattern was unintentional. Changed to `self.user.is_superuser = True` instead.